validation/validation_test.go

```diff
@@ -88,13 +88,53 @@ func TestValidateBasic(t *testing.T) {
 	assert.Nil(t, runtimeInsideValidate(g))
 }
 
+// Test whether rootfs Readonly can be applied as false
+func TestValidateRootFS(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetRootReadonly(false)
+
+	assert.Nil(t, runtimeInsideValidate(g))
+}
+
+// Test whether rootfs Readonly can be applied as true
+func TestValidateRootFS(t *testing.T) {
+	g := getDefaultGenerator()
+	g.SetRootReadonly(true)
```

Comment:
I think we need to skip this on Windows, where [the spec forbids setting `readonly` to `true`][1].  Also, you probably need to use more-specific function names:

```
$ git grep '^func TestValidateRootFS' origin/pr/512
origin/pr/512:validation/validation_test.go:func TestValidateRootFS(t *testing.T) {
origin/pr/512:validation/validation_test.go:func TestValidateRootFS(t *testing.T) {
```

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config.md#L46